test/test_retryable_writes.py

```diff
@@ -453,6 +454,27 @@ def test_batch_splitting_retry_fails(self):
             self.assertEqual(final_txn, expected_txn)
         self.assertEqual(coll.find_one(projection={'_id': True}), {'_id': 1})
 
+    @client_context.require_version_min(4, 4)
+    def test_retryable_write_error_label_is_propagated(self):
+        listener = OvertCommandListener()
+        client = rs_or_single_client(
+            retryWrites=True, event_listeners=[listener])
```

Comment:
Can we require the client_context client instead of creating a new one?

Reply:
How do I add an event listener to the existing client?